logbook-netty/src/test/java/org/zalando/logbook/netty/LogbookClientHandlerTest.java

```diff
@@ -48,11 +44,8 @@
             )
             .bindNow();
 
-    @SuppressWarnings("deprecation") // the alternative method is not available prior to 1.0.0
     private final HttpClient client = HttpClient.create()
-            .tcpConfiguration(tcpClient ->
-                    tcpClient.doOnConnected(connection ->
-                            connection.addHandlerLast(new LogbookClientHandler(logbook))))
+            .doOnConnected(connection -> connection.addHandlerLast(new LogbookClientHandler(logbook)))
```

Comment:
The suppression was important, because the replacement method breaks backwards compatibility with prior versions of netty.

Reply:
OK, I've reverted